gateway/cli/src/main.rs

```diff
@@ -22,6 +25,13 @@ struct Cli {
 
 #[derive(Subcommand)]
 pub enum Commands {
+    /// Ganerate gateway configuration
+    /// NOTE: This command can only be used on a local gateway
+    GenerateConfig {
+        /// The gateway configuration directory
+        #[clap(default_value = ".gateway")]
+        out_dir: PathBuf,
+    },
```

Comment:
For which component is this cfg? The client or the gateway?

Reply:
The gateway

Reply:
I added this helper to the gateway-cli so you can quickly gen a config for your local GW. Ideal use cases are:
- you generate a config and copy them over to some remote GW deployment
- you generate config into a known dir, which you can pipe to your local gateway when starting it